### ISSUE TITLE
Job Results - Output

### ISSUE THREAD

**Issue by mabashian** [ansible/awx#4224]
## Job Results - Output

Mockup: DNE, see yolotower

The work required here will likely be significant but the UI itself is fairly straightforward so I haven't broken it out into a bunch of pieces.  We should look at the existing awx logic in this area and see if we can re-use some of the same code/patterns to cut down on development time and avoid creating the wheel again.

Note that the first task is _temporary_ so we shouldn't spend a ton of time working on that because it will be thrown away.

#### Tasks
* Display loading icon/text while job is running (this will require websockets or polling for "real time")
* Display static output rows once job is complete (scrollable and fetches more rows dynamically)

#### Not in Scope for MVP
* Expand/collapse play/task
* Up/down arrows for navigating output
* Back to top shortcut
* Event tooltip/popover when clicked
* Live event loading (websockets)

Example from yolotower (use this as a content reference not an actual mockup):

<img width="942" alt="Screen Shot 2019-05-23 at 11 12 38 AM" src="https://user-images.githubusercontent.com/9889020/58264347-b4273b80-7d4b-11e9-9ea3-7083763bdf5e.png">

**Comment by mabashian** [comment]
Blocked until https://github.com/ansible/awx/issues/4236 lands

**Comment by jakemcdermott** [comment]
see: https://github.com/ansible/awx/pull/4383

Most of this is done and is no longer blocking https://github.com/ansible/awx/issues/4430

I'm closing this issue, we'll track the remaining items in a new issue, which I've added to the backlog: https://github.com/ansible/awx/issues/4477
